deploy/helm-chart/templates/rbac.yaml

```diff
@@ -7,6 +7,19 @@ metadata:
   {{- include "nginx-gateway.labels" . | nindent 4 }}
   annotations:
     {{- toYaml .Values.serviceAccount.annotations | nindent 4 }}
+{{- if or .Values.serviceAccount.imagePullSecretName .Values.serviceAccount.imagePullSecretsNames }}
+imagePullSecrets:
+{{- end }}
```

Comment:
This end can just be at the end of all the if statements. I would nest them inside this top level one with some indentations.

There's also a lot of spacing in here (extra lines) that we can reduce.

Reply:
updated this